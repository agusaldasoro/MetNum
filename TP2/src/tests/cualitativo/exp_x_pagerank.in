0 0.85 0 tests/cualitativo/exp_x_res.out 0.00001 
