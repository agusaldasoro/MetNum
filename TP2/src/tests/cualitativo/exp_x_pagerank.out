0.107686
0.218026
0.159032
0.096085
0.021888
0.048616
0.040098
0.013623
0.040098
0.013623
0.013623
0.048363
0.013623
0.013623
0.013623
0.015939
0.064868
0.057563
